standard/wallets/restricted.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Restricted Wallet"
+---
+
+import { Aside } from '/snippets/aside.jsx';
+
+Restricted Wallet is a specialized wallet contract that restricts transfers to only one predefined destination address.
+
+## What makes it different
+
+Unlike standard wallets that allow transfers to any address, Restricted Wallet implements address-based restrictions that limit where funds can be sent.
+
+**Standard wallet behavior:**\
+You can send funds to any valid address.
+
+**Restricted Wallet behavior:**\
+You can only send funds to the predefined destination address (except for validation contracts).
```

Comment:
Explain what is "validation contracts". Maybe we need to say something like this: restricted wallet can only send funds to the owner of this wallet, to Elector with specified opcodes (describe them in short, refer to Elector doc), and to Config with specified opcode (describe it and link to Config doc)